#! /bin/sh /usr/share/dpatch/dpatch-run
## 11_remove_postproc_support_from_modules-stream_out-switcher.c.dpatch by Daniel T Chen <crimsun@fungus.sh.nu>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Hack around libpostproc-dev/multiverse crackfulness by removing 
## DP: postproc support from modules/stream_out/switcher.c

@DPATCH@

--- vlc.1/modules/stream_out/switcher.c	2005-10-11 03:44:47.000000000 -0700
+++ vlc/modules/stream_out/switcher.c	2005-10-11 03:44:58.000000000 -0700
@@ -41,9 +41,8 @@
 #   include <avcodec.h>
 #endif
 
-#ifdef HAVE_POSTPROC_POSTPROCESS_H
+#if 0
 #   include <postproc/postprocess.h>
-#else
 #   include <libpostproc/postprocess.h>
